<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FresnelChartDialog</class>
 <widget class="QWidget" name="FresnelChartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>644</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dettagli riflettore di vetro</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>660</width>
       <height>490</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="2">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>660</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>2</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="positioningLabel">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {background-color: #fafafa}
QLabel {color: #000000}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="refractionLabel">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {background-color: #fafafa}
QLabel {color: #000000}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="hazardLabel">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {background-color: #fafafa}
QLabel {color: #000000}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="DNRO_Label">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {background-color: #fafafa}
QLabel {color: #000000}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="distanceLabel">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {background-color: #fafafa}
QLabel {color: #000000}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>7</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}
QLabel {margin-left: 10px}</string>
          </property>
          <property name="text">
           <string>Posizionamento [gradi]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>7</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}
QLabel {margin-left: 10px}</string>
          </property>
          <property name="text">
           <string>Indice rifrazione</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>7</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}
QLabel {margin-left: 10px}</string>
          </property>
          <property name="text">
           <string>Distanza di rischio [m]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>7</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}
QLabel {margin-left: 10px}</string>
          </property>
          <property name="text">
           <string>DNRO [m]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>7</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}
QLabel {margin-left: 10px}</string>
          </property>
          <property name="text">
           <string>Distanza da laser [m]</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
